art/attacks/evasion/projected_gradient_descent/projected_gradient_descent.py

```diff
@@ -178,8 +178,8 @@ def set_params(self, **kwargs) -> None:
 
     def _check_params(self) -> None:
         # Check if order of the norm is acceptable given current implementation
-        if self.norm not in [np.inf, int(1), int(2)]:
-            raise ValueError("Norm order must be either `np.inf`, 1, or 2.")
+        if self.norm not in ["inf", np.inf, int(1), int(2)]:
```

Comment:
For consistency:
if self.norm not in [1, 2, np.inf, "inf"]:

Reply:
Fixed.